### PR TITLE
Add copyright headers

### PR DESCRIPTION
**Pull request by annp1987**
label: 'release notes: no'

### CODE REVIEW

**Comment by annp1987** [comment]
Hi Everyone,

I'm a newbie contributor. I don't know how to add release note label. Could you please teach me? I really appreciate your teaching.

**Comment by lidizheng** [comment]
@hsaliak

File: tools/http2_interop/doc.go [hunk 1]
```diff
@@ -1,3 +1,17 @@
+// Copyright 2019 gRPC authors.
```
Comment (jtattermusch) [inline tools/http2_interop/doc.go]: The new format is `Copyright 2019 The gRPC Authors`
as seen e.g. here: https://github.com/grpc/grpc/blob/906c12568e9bdf1d8306a2f2ab45efe8beb43f74/src/csharp/Grpc.Core.Api/Properties/AssemblyInfo.cs#L3

**Comment by jtattermusch** [comment]
> Hi Everyone,
> 
> I'm a newbie contributor. I don't know how to add release note label. Could you please teach me? I really appreciate your teaching.

only grpc maintainers can add labels.

File: tools/http2_interop/doc.go (hunk 1 above)
Comment (annp1987) [reply tools/http2_interop/doc.go]: Thank you!

File: tools/http2_interop/doc.go [hunk 2]
```diff
@@ -1,3 +1,17 @@
+// Copyright 2019 The gRPC Authors.
```
Comment (jtattermusch) [inline tools/http2_interop/doc.go]: no dot at the end of the line

Comment (annp1987) [reply tools/http2_interop/doc.go]: Thanks. Done.

**Comment by jtattermusch** [comment]
Comment-only change, merging.

**Pull request merged**